pkgs/development/python-modules/zetup/default.nix

```diff
@@ -11,11 +11,10 @@ buildPythonPackage rec {
     sha256 = "b8a9bdcfa4b705d72b55b218658bc9403c157db7b57a14158253c98d03ab713d";
   };
 
-  # Python 3.7 compatibility
-  # See https://github.com/zimmermanncode/zetup/pull/1
+  # Python 3.8 compatibility
   postPatch = ''
     substituteInPlace zetup/zetup_config.py \
-      --replace "'3.6']" "'3.6', '3.7']"
+      --replace "'3.7']" "'3.7', '3.8']"
```

Comment:
this kind of change could be upstreamed

Reply:
that would be https://github.com/zimmermanncode/zetup/blob/master/zetuprc#L11

Reply:
I was actually staring at it for a while.  What's weird is the file I'm patching isn't the `zetuprc` (plain text), it's the `zetup/zetup_config.py`, which appears to be code-gen'd from the `zetuprc` (?) and doesn't exist in the repo.  I've sent a PR tho that I think will do it; we'll see what the maintainer says.

Reply:
Thanks :+1: